cset-workflow/includes/lfric_deterministic_domain_mean_surface_time_series.cylc

```diff
@@ -12,6 +12,11 @@
         --SUBAREA_LAT_BOUND_TOP={{SUBAREA_LAT_BOUND_TOP}}
         --SUBAREA_LON_BOUND_RIGHT={{SUBAREA_LON_BOUND_RIGHT}}
         --SUBAREA_LON_BOUND_LEFT={{SUBAREA_LON_BOUND_LEFT}}
+        {% else %}
+        --SUBAREA_LAT_BOUND_BOTTOM="None"
+        --SUBAREA_LAT_BOUND_TOP="None"
+        --SUBAREA_LON_BOUND_RIGHT="None"
+        --SUBAREA_LON_BOUND_LEFT="None"
```

Comment:
```suggestion
        --SUBAREA_LAT_BOUND_BOTTOM=None
        --SUBAREA_LAT_BOUND_TOP=None
        --SUBAREA_LON_BOUND_RIGHT=None
        --SUBAREA_LON_BOUND_LEFT=None
```
If we drop the quotes does it parse it as a None singleton? I think the issues was that this never actually goes through a shell, and thus it doesn't get unquoted.

Reply:
Will try if this works now